test/net-test.js

```diff
@@ -1714,7 +1821,7 @@ describe('Net', function() {
 
         const pkt = new packets.GetDataPacket([
           new InvItem(InvItem.types.WITNESS_FILTERED_BLOCK,
-                      Buffer.alloc(32, 0x01))
+            Buffer.alloc(32, 0x01))
```

Comment:
For the rest of these changes, did your IDE make this lint changes automatically? Is it possible to ignore them for now?